### PR TITLE
Add collapse shortcode use cases

### PR DESCRIPTION
**Pull request by imatwawana**
Constrains use cases for the `collapse` shortcode to deprecations and configuration options per the Oct 5 style roundtable meeting. Also adds guidance re lead-in sentences and titles.

### CODE REVIEW

File: docs/sources/write/shortcodes/index.md [hunk 1]
```diff
@@ -128,12 +128,77 @@ Produces:
...
 
+### Deprecation example
+
+```
+#### BoltDB (deprecated)
+
+The following example is for a deprecated store and shouldn't be used for new Loki deployments:
+
+{{</* collapse title="boltdb-shipper" */>}}
+Also known as “boltdb-shipper” during development (and is still the schema store name).
```
Comment (osg-grafana) [inline docs/sources/write/shortcodes/index.md:152]:
```suggestion
Also known as _boltdb-shipper_ during development (and is still the schema store name).
```

File: docs/sources/write/shortcodes/index.md [hunk 2]
```diff
@@ -128,12 +128,77 @@ Produces:
...
+
+Produces:
+
+#### BoltDB (deprecated)
+
+The following example is for a deprecated store and shouldn't be used for new Loki deployments:
+
+{{< collapse title="boltdb-shipper" >}}
+Also known as “boltdb-shipper” during development (and is still the schema store name). 
+The single store configurations for Loki utilize the chunk store for both chunks and the index, requiring just one store to run Loki.
```
Comment (osg-grafana) [inline docs/sources/write/shortcodes/index.md:165]:
```suggestion
The single store configurations for Loki to utilize the chunk store for both chunks and the index, which requires just one store to run Loki.
```

File: docs/sources/write/shortcodes/index.md [hunk 3]
```diff
@@ -128,12 +128,77 @@ Produces:
...
+  compaction_interval: 5m
+```
+{{</* /collapse */>}}
+````
+
+Produces:
+
+#### 6-Compactor-Snippet.yaml
+
+The following partial configuration sets the compactor to use S3 and run the compaction every 5 minutes.
```
Comment (osg-grafana) [inline docs/sources/write/shortcodes/index.md:190]:
```suggestion
The following partial configuration sets the compactor to use Amazon S3 and runs the compaction every 5 minutes.
```

**Review by osg-grafana — APPROVED** [review]
Unblocking with feedback

File: docs/sources/write/shortcodes/index.md [hunk 4]
```diff
@@ -128,12 +128,77 @@ Produces:
...
 You can't do the following with this shortcode:
 
-- Use these as page headings
+- Use them as page headings
 - Control the size of the title text
 - Add images or videos between the shortcode tags
 
+### Deprecation example
+
+```
```
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:146]:
````suggestion
```markdown
````

File: docs/sources/write/shortcodes/index.md [hunk 5]
```diff
@@ -128,12 +128,77 @@ Produces:
...
+The following example is for a deprecated store and shouldn't be used for new Loki deployments:
+
+{{< collapse title="boltdb-shipper" >}}
+Also known as “boltdb-shipper” during development (and is still the schema store name). 
+The single store configurations for Loki utilize the chunk store for both chunks and the index, requiring just one store to run Loki.
+{{< /collapse >}}
+
+### Configuration options example
+
+````
```
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:170]:
`````suggestion
````markdown
`````

**Review by JStickler — APPROVED** [review]
Nice!  Thanks for writing this up so clearly.

File: docs/sources/write/shortcodes/index.md [hunk 6]
```diff
@@ -128,12 +128,79 @@ Produces:
...
+```
+
+Produces:
+
+#### BoltDB (deprecated)
+
+The following example is for a deprecated store and shouldn't be used for new Loki deployments:
+
+{{< collapse title="boltdb-shipper" >}}
+Also known as “boltdb-shipper” during development (and is still the schema store name).
```
Comment (imatwawana) [inline docs/sources/write/shortcodes/index.md:164]:
```suggestion
Also known as _boltdb-shipper_ during development (and is still the schema store name).
```

**Pull request merged**